start.bash

```diff
@@ -11,13 +11,20 @@
 # WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
 # See the License for the specific language governing permissions and
 # limitations under the License.
+if [[ "$(whoami)" != "root" ]]; then
+    echo "Please change to root user and retry."
+    exit 1
+fi
 
-echo "Docker pull SQLFlow images ..."
+echo "Docker pull dependency images, you can comment this if already have them ..."
 # c.f. https://github.com/sql-machine-learning/sqlflow/blob/develop/.travis.yml
 docker pull sqlflow/sqlflow:jupyter
 docker pull sqlflow/sqlflow:mysql
 docker pull sqlflow/sqlflow:server
 docker pull sqlflow/sqlflow:step
+docker pull argoproj/argoexec:v2.7.7
```

Comment:
Should we cache these images as lz4 files on our mirror sites? So we can docker import them.

Reply:
Yes, actually, I'm doing this when I playing locally.